bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -482,14 +432,14 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
      simplicity, we tie the defaults for both of these options to the overall
      CLM phys version, rather than having some intermediate option that controls
      the defaults for both h2osno_max and glc_snow_persistence_max_days. -->
-<glc_snow_persistence_max_days phys="clm5_1" >0</glc_snow_persistence_max_days>
-<glc_snow_persistence_max_days phys="clm5_0" >0</glc_snow_persistence_max_days>
+<glc_snow_persistence_max_days               >0</glc_snow_persistence_max_days>
 <glc_snow_persistence_max_days phys="clm4_5" >7300</glc_snow_persistence_max_days>
 
 <!-- ================================================================== -->
 <!-- The default filenames are given relative to the root directory
      for the CLM2 data in the CESM distribution -->
 <!-- Plant function types (relative to {csmdata}) -->
+<paramfile phys="clm6_0" >lnd/clm2/paramdata/ctsm51_params.c240208.nc</paramfile>
```

Comment:
I'll copy this to an identical one that has ctsm60 in the name. Later when we update the ctsm51 version can be removed in the file. For future tuning on the ctsm60 will be tuned.